fbw-a32nx/src/systems/instruments/src/PFD/LandingSystemIndicator.tsx

```diff
@@ -741,3 +747,59 @@ class MarkerBeaconIndicator extends DisplayComponent<{ bus: ArincEventBus }> {
     );
   }
 }
+
+class LsReminderIndicator extends DisplayComponent<{ bus: ArincEventBus }> {
+  private readonly sub = this.props.bus.getSubscriber<PFDSimvars & FcuBus & FgBus>();
+
+  private readonly glsMlsFlsOrLocVnavInstalled = Subject.create(false);
+
+  private readonly fwcFlightPhase = ConsumerSubject.create(this.sub.on('fwcFlightPhase'), 0);
+
+  private readonly fmgcFlightPhase = ConsumerSubject.create(this.sub.on('fmgcFlightPhase'), FmgcFlightPhase.Preflight);
+
+  private readonly fmgcDiscreteWord2 = Arinc429ConsumerSubject.create(this.sub.on('fmgcDiscreteWord3'));
+
+  private readonly fmgcDiscreteWord4 = Arinc429ConsumerSubject.create(this.sub.on('fmgcDiscreteWord4'));
+
+  private readonly landModeArmedOrActive = MappedSubject.create(
```

Comment:
I think the condition should also include LOC modes (armed/capture/track) to account for the case when flying a LOC only approach